bliss/__init__.py

```diff
@@ -0,0 +1,10 @@
+# flake8: noqa
+import omegaconf
+
+
+def make_range(start, stop, step):
+    return omegaconf.ListConfig(list(range(start, stop, step)))
+
+
+# resolve ranges in config files (we want this to execute at any entry point)
+omegaconf.OmegaConf.register_new_resolver("range", make_range, replace=True)
```

Comment:
Is there anywhere else this command could go? I'm not sure where, but I'm concerned that new developers on the project won't find it if it's here, and therefore won't know about it.

Reply:
I'm not really a huge fan of this either, but I'm not sure where else to put it. Currently we have three entry points to bliss - one in main.py, one from the API functions in api.py, and one in the tests. Instead of duplicating it in each of those locations, just having it in the top-level `__init__.py` ensures that it's loaded no matter the entry point, but does run the risk of it being overlooked.

An alternate option could be to put it in its own script, like `bliss/setup.py`, and import that in `__init__.py` instead. It still requires putting something in the init, but it would be just one import instead of additional logic. Do you think that's a better idea?

Reply:
I guess it's OK here after all.

Reply:
Adding a new file with so little code probably wouldn't simplified things